ibis/expr/types.py

```diff
@@ -731,6 +731,9 @@ class SortExpr(Expr):
     def _type_display(self):
         return 'array-sort'
 
+    def get_name(self):
```

Comment:
Is this name propagation only used for Sort expressions? It suggests that somewhere We don't have a clear distinction between expressions and operations.

Reply:
No, `get_name` is a method on `Expr`.

Reply:
I'll change to not use `resolve_name` and make an issue to move `resolve_name` into `Node` (instead of just in `ValueOp` where it is now).

Reply:
Created #1552 to track this.